en/components/card.md

```diff
@@ -287,8 +287,122 @@ You can also justify the buttons so that they are layed out across the entire ax
 </igx-card-actions>
 ```
 
-#### Summary
-In this article we covered a lot of ground with the card component. First, we created a very simple card with text content only. Then added some images to make the card a bit more appealing. Finally, we used some additional Ignite UI for Angular components inside our card, like avatars, buttons and icons, to enrich the experience and add some functionality. The card component is capable of displaying more different layouts worth exploring in the Card Demo in the beginning of this article.
+### Styling
```

Comment:
We need to update the **jp** version of the file as well.

Reply:
JP file updated.